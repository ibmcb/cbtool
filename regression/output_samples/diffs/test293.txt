 status: Sending a termination request for vm_55 (cloud-assigned uuid A66C88F3-9AA7-5A13-8D69-B8F9E5CDB8CF)....
 status: Sending a termination request for vm_57 (cloud-assigned uuid 1A5D210E-742D-5C25-8687-DF29BC8A46B7)....
 status: Waiting for vm_56 (cloud-assigned uuid DCAF56CC-4D33-5B78-97CF-E6C212B757FA) to be captured with image name "vm_56_at_1377115657"...
 status: Sending a termination request for vm_56 (cloud-assigned uuid DCAF56CC-4D33-5B78-97CF-E6C212B757FA)....
One of the VMs ("vm_56") belonging to the AI object 42CF13A2-32F2-5B4B-B1B8-94D8B8866A4D (named "ai_10") successfully captured on this experiment.
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |197.241.39.37   |1377115468     |none                                    |07AC899E-55E1-5C90-BE3A-8BEBC45DC18B
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |54.30.255.166   |1377115485     |none                                    |1C0B4DFB-2660-597B-9885-42A17F481679
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |223.151.68.176  |1377115570     |none                                    |CE583588-D25B-5D07-9122-4E260F85EB45
|ai_9      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |234.122.201.235 |1377115574     |none                                    |4CF77846-E422-56F1-942A-0538BDD90BFC
|ai_11     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |82.53.142.62    |1377115584     |none                                    |DBC62C54-4E8C-5366-B164-0B66C5DA33A1
|ai_14     |netperf        |1_x_netclient->1_x_netserver                      |91.72.211.172   |1377115600     |none                                    |FADE3E04-A0B5-53B8-89D4-09FA2A832149
|ai_15     |netperf        |1_x_netclient->1_x_netserver                      |35.179.152.22   |1377115602     |none                                    |5B557639-6184-53AE-9A2C-09621D23F1A1
|ai_16     |netperf        |1_x_netclient->1_x_netserver                      |116.38.175.124  |1377115604     |none                                    |C3D53D7C-0AE7-5855-9219-8A17E5AAB41D
|ai_17     |netperf        |1_x_netclient->1_x_netserver                      |102.230.196.9   |1377115606     |none                                    |E64F773A-F7FB-5686-A643-565EF46639FC
|ai_18     |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |102.255.129.159 |1377115613     |none                                    |BA8546BB-EA00-5F8F-99F4-AA58BAEE2A1E
|ai_19     |tradelite      |1_x_client_tradelite->1_x_was                     |98.170.233.217  |1377115618     |none                                    |388D5EEC-BCD2-542D-9518-91EDA4514265
|ai_20     |hpcc           |1_x_fen_hpc->3_x_cn_hpc                           |81.69.239.191   |1377115623     |none                                    |5750EDE7-8F2C-5563-9178-EF3EB6E41DD3
|ai_21     |coremark       |1_x_coremark                                      |185.58.19.49    |1377115628     |none                                    |2F4D6D36-BD36-5F23-9F4F-FB79C4F3704B
|ai_23     |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |202.142.233.244 |1377115646     |none                                    |61AA42EB-5AD4-5106-9409-F3E93E5E3BE2
|ai_24     |netperf        |1_x_netclient->1_x_netserver                      |129.204.186.67  |1377115650     |none                                    |25BA5105-A046-50FF-868E-E60D5EDB11FB
|ai_25     |hpcc           |1_x_fen_hpc->3_x_cn_hpc                           |255.65.29.140   |1377115655     |none                                    |47A2ADC2-9EB9-5B0B-84FD-EDA7148B18FC
|vm_36         |client_daytrader    |nano32      |197.241.39.37   |simhostc1                  |SUT                 |ai_4    |none       |BC108A18-3B66-576E-8895-707EF69F7861
|vm_38         |db2                 |platinum64  |200.122.223.254 |simhostc1                  |SUT                 |ai_4    |none       |D34D700A-54A2-5415-8F14-E87E0E5DF352
|vm_37         |was                 |iron32      |14.61.28.112    |simhostc2                  |SUT                 |ai_4    |none       |68B954F9-0EA7-5BDE-9FBA-96DC72A282AC
|vm_40         |netserver           |iron32      |236.2.66.133    |simhostb3                  |SUT                 |ai_5    |none       |65EDE7C1-CD8A-5CBD-B129-7861E35B8DB6
|vm_39         |netclient           |platinum64  |54.30.255.166   |simhostc3                  |SUT                 |ai_5    |none       |279A40B7-0AA1-57B3-82B5-11ABBCC19914
|vm_47         |lb                  |iron32      |6.153.56.98     |simhosta1                  |SUT                 |ai_8    |none       |95FA1F88-D813-5680-8095-A3C1AF463C3A
|vm_48         |client_daytrader    |nano32      |223.151.68.176  |simhostb2                  |SUT                 |ai_8    |none       |F91D82D0-4443-5114-B568-23754138AB99
|vm_49         |was                 |iron32      |164.134.83.36   |simhosta1                  |SUT                 |ai_8    |none       |B6490274-CA3A-5E0E-9DB9-8BCA5E446B61
|vm_50         |was                 |iron32      |212.230.70.14   |simhostc3                  |SUT                 |ai_8    |none       |D1F3260F-632D-5B5D-A920-F352908BFDF1
|vm_51         |db2                 |gold32      |197.16.122.35   |simhostb0                  |SUT                 |ai_8    |none       |ACBEFE1E-9585-5A3F-8BC0-42C34B32A7D0
|vm_52         |db2                 |platinum64  |56.30.143.201   |simhosta4                  |SUT                 |ai_9    |none       |D835B241-C6FF-5635-8006-DE009C3DD0C5
|vm_53         |was                 |iron32      |205.104.166.7   |simhostc0                  |SUT                 |ai_9    |none       |00EBBA5F-C1DB-527C-97E1-C9C12F098AA0
|vm_54         |client_daytrader    |nano32      |234.122.201.235 |simhosta1                  |SUT                 |ai_9    |none       |FF9C9F38-2A74-5FAA-99D2-D13A768CE94D
|vm_58         |client_daytrader    |nano32      |82.53.142.62    |simhostc2                  |SUT                 |ai_11   |none       |9629F384-29DF-5A7E-8C4E-8231683D2CDD
|vm_59         |was                 |iron32      |224.31.240.136  |simhostc2                  |SUT                 |ai_11   |none       |8E4C8A36-8E3F-5DEF-95F6-72E869BB2D01
|vm_60         |db2                 |platinum64  |56.55.134.162   |simhosta3                  |SUT                 |ai_11   |none       |7B09295B-5CE3-532D-9D05-2BB37A1EA526
|vm_71         |netclient           |iron32      |91.72.211.172   |simhostd2                  |LG                  |ai_14   |none       |9B203B56-F368-57E8-87FE-7E41F45E1C3C
|vm_72         |netserver           |iron32      |74.242.127.136  |simhosta3                  |SUT                 |ai_14   |none       |130B5168-2B4C-52ED-9E8D-7C99E2CD8CA9
|vm_73         |netclient           |iron32      |35.179.152.22   |simhostd2                  |LG                  |ai_15   |none       |17EE8E3A-0825-5EBC-A8A6-E1132BF4BF31
|vm_74         |netserver           |iron32      |84.111.3.4      |simhosta1                  |SUT                 |ai_15   |none       |530B266D-58F9-51CD-B644-A68BB5F22D2F
|vm_75         |netclient           |iron32      |116.38.175.124  |simhostd0                  |LG                  |ai_16   |none       |4C3596A6-5376-5568-81BB-67AFE129E732
|vm_76         |netserver           |iron32      |121.135.163.129 |simhostb4                  |SUT                 |ai_16   |none       |2FEED946-6869-528A-B14F-13E3B5ECB4C5
|vm_77         |netclient           |iron32      |102.230.196.9   |simhostd1                  |LG                  |ai_17   |none       |61D5BF09-047F-5FA5-87EF-292D0F62EE7B
|vm_78         |netserver           |iron32      |227.180.39.141  |simhostc1                  |SUT                 |ai_17   |none       |A45D8502-4D8D-5E84-97A2-D128536DC676
|vm_79         |client_daytrader    |nano32      |102.255.129.159 |simhosta0                  |SUT                 |ai_18   |none       |E7109880-9119-5BD0-ACD9-8AE96FA9905B
|vm_80         |lb                  |iron32      |35.168.107.20   |simhosta2                  |SUT                 |ai_18   |none       |FF977E9B-D7B6-5F12-93D8-3ADBD53A8B05
|vm_81         |was                 |iron32      |158.226.169.48  |simhostc3                  |SUT                 |ai_18   |none       |2E2BE497-1A72-51F4-9038-49D1163C707D
|vm_83         |db2                 |gold32      |44.54.66.36     |simhostc2                  |SUT                 |ai_18   |none       |48583E08-767C-58DF-8729-F78428222874
|vm_82         |was                 |iron32      |6.152.56.115    |simhostb1                  |SUT                 |ai_18   |none       |1FF77042-875D-5B17-A81A-488990EFE8D3
|vm_85         |client_tradelite    |nano32      |98.170.233.217  |simhostd0                  |LG                  |ai_19   |none       |70AF3F52-4AFB-5CD8-AB86-B24520B2183A
|vm_84         |was                 |iron32      |68.49.228.118   |simhostc4                  |SUT                 |ai_19   |none       |F88D57F1-D758-51E2-9ACA-7DEB4770FBDE
|vm_86         |fen_hpc             |bronze32    |81.69.239.191   |simhostc4                  |SUT                 |ai_20   |none       |686C1047-1AD3-5070-ACAE-8AE8FFAEC88A
|vm_87         |cn_hpc              |bronze32    |124.117.139.184 |simhosta4                  |SUT                 |ai_20   |none       |806AC1BB-B971-501C-8A0F-FCEA2E320059
|vm_88         |cn_hpc              |bronze32    |238.33.23.241   |simhosta3                  |SUT                 |ai_20   |none       |F4F42A16-E8F8-55CC-AA35-F32FD086E3F9
|vm_89         |cn_hpc              |bronze32    |72.232.183.29   |simhostb4                  |SUT                 |ai_20   |none       |1335D249-40A4-559F-879C-C4CD7648FD05
|vm_90         |coremark            |iron32      |185.58.19.49    |simhosta2                  |SUT                 |ai_21   |none       |E7C19E26-72E1-570D-B6EB-7989698B94AA
|vm_92         |client_daytrader    |nano32      |202.142.233.244 |simhostb4                  |SUT                 |ai_23   |none       |D1CD439E-49D2-5D9A-B6DF-9AF21EF602F1
|vm_94         |was                 |iron32      |182.247.109.93  |simhostb0                  |SUT                 |ai_23   |none       |8307B97A-E278-5CBC-9000-7624A3A6DAED
|vm_93         |lb                  |iron32      |176.156.239.163 |simhostc4                  |SUT                 |ai_23   |none       |39773F3B-BC53-569B-9AB6-251F43F93202
|vm_95         |was                 |iron32      |63.50.159.218   |simhostb4                  |SUT                 |ai_23   |none       |1A36E720-6657-50DA-A8B1-2CB080408FF2
|vm_96         |db2                 |gold32      |215.196.208.16  |simhosta0                  |SUT                 |ai_23   |none       |9DF2E30B-9F72-5A2A-BFE7-3A73D8857E70
|vm_97         |netclient           |iron32      |129.204.186.67  |simhostd0                  |LG                  |ai_24   |none       |7899E826-5363-5B35-973E-78BCB670B328
|vm_98         |netserver           |iron32      |124.143.26.81   |simhostb1                  |SUT                 |ai_24   |none       |65DCA7F5-2497-50C2-9FDA-A837022CB002
|vm_99         |fen_hpc             |bronze32    |255.65.29.140   |simhostb3                  |SUT                 |ai_25   |none       |2C525495-11D4-54E1-B523-3B1C74BFEB49
|vm_100        |cn_hpc              |bronze32    |199.239.69.76   |simhostc1                  |SUT                 |ai_25   |none       |8F23E988-960B-5E78-B19E-31BB04C594DD
|vm_101        |cn_hpc              |bronze32    |23.240.203.131  |simhostc1                  |SUT                 |ai_25   |none       |88DE78E0-BD2B-5675-8655-928D24364B80
|vm_102        |cn_hpc              |bronze32    |217.164.20.126  |simhostc1                  |SUT                 |ai_25   |none       |D902D2EE-E539-5904-9799-7FA4B526A7ED
|Used Memory                                          |22.58M                        
|Uptime (in seconds)                                  |2516781                       
|Total Connections Received                           |185669                        
|Total Commands Processed                             |11709636                      
|Number of Keys                                       |880                           
|Storage Size                                         |18579456                      
|Data Size                                            |6805820                       
|Index Size                                           |2133936                       
|Average Object Size                                  |616.972169341                 
|AD93E38D-EAEC-5C15-BA20-7A27F63D13FF (simzone_b) VM RESERVATIONS|11                            
|E4B581D8-2DD3-5956-8196-B1B60C5DC5B0 (simzone_d) VM RESERVATIONS|6                             
|AD29082C-1389-58A3-BE64-3784D075F191 (simzone_c) VM RESERVATIONS|17                            
|75A875C6-F2C5-5C3D-BD34-051FBBE24C3A (simzone_a) VM RESERVATIONS|13                            
