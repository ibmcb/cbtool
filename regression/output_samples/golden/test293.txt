 status: Waiting for vm_84 (cloud-assigned uuid C0F84CEA-F9DA-54B2-9ABF-ECE31FEAEA9C) to start...
 status: Waiting for vm_86 (cloud-assigned uuid 2567B2C1-CA4B-5270-B592-BB82E572AF6C) to start...
 status: Waiting for vm_85 (cloud-assigned uuid 07042BA5-FC74-5790-A82F-B64F6D724C77) to start...
 status: Waiting for vm_87 (cloud-assigned uuid 23001BF0-9A97-59AF-9E33-539FD8BB670A) to start...
 status: Trying to establish network connectivity to vm_86 (cloud-assigned uuid 2567B2C1-CA4B-5270-B592-BB82E572AF6C), on IP address 73.215.44.132...
 status: Trying to establish network connectivity to vm_85 (cloud-assigned uuid 07042BA5-FC74-5790-A82F-B64F6D724C77), on IP address 14.225.133.238...
 status: Trying to establish network connectivity to vm_84 (cloud-assigned uuid C0F84CEA-F9DA-54B2-9ABF-ECE31FEAEA9C), on IP address 79.143.111.53...
 status: Trying to establish network connectivity to vm_87 (cloud-assigned uuid 23001BF0-9A97-59AF-9E33-539FD8BB670A), on IP address 190.118.85.179...
 status: Bypassing the sending of a copy of the code tree to vm_85 (14.225.133.238)...
 status: Bypassing the sending of a copy of the code tree to vm_87 (190.118.85.179)...
 status: Bypassing the sending of a copy of the code tree to vm_84 (79.143.111.53)...
 status: Bypassing the sending of a copy of the code tree to vm_86 (73.215.44.132)...
 status: Bypassing generic VM post_boot configuration.
 status: Bypassing application-specific "setup" operations
AI object F78136C2-A67C-5D2C-8A6E-C67EDBD28789 (named "ai_20") sucessfully attached to this experiment. It is ssh-accessible at the IP address 79.143.111.53 (C0F84CEA-F9DA-54B2-9ABF-ECE31FEAEA9C.simcloud.com).
The following AIs are attached to this experiment (Cloud TESTCLOUD) :
|name      |type           |sut                                               |cloud_ip        |arrival        |aidrs                                   |uuid
|ai_3      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |141.98.169.121  |1358912185     |none                                    |CECA49F8-7F0A-5A2A-88AB-6FBD3C13A188
|ai_4      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |20.62.246.201   |1358912222     |none                                    |AA2AF411-5127-5D04-935C-861E9991D9CD
|ai_5      |netperf        |1_x_netclient->1_x_netserver                      |88.182.228.241  |1358912240     |none                                    |22D578AD-237A-5FD0-BF2E-8526B27365A8
|ai_8      |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |120.99.243.188  |1358912324     |none                                    |DA622B54-93C1-584C-A13C-A7741CB76ADD
|ai_9      |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |88.118.248.29   |1358912327     |none                                    |DA3C582D-82E4-5D44-87E8-6543B22AB272
|ai_10     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |112.27.118.248  |1358912330     |none                                    |CEE9035A-0ECD-5ACA-AC70-99DCCEA9A76C
|ai_11     |daytrader      |1_x_client_daytrader->1_x_was->1_x_db2            |127.64.103.51   |1358912333     |none                                    |E5875E74-EFDB-546D-80B6-D5BE50F7D53B
|ai_12     |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |192.201.187.120 |1358912338     |none                                    |C536D81E-D85D-57E3-A40C-C6B2A7AD05C6
|ai_13     |daytrader      |1_x_client_daytrader->1_x_lb->2_x_was->1_x_db2    |243.19.9.98     |1358912344     |none                                    |EB504B39-1AC5-5B0B-9D6B-7704E588E2DE
|ai_14     |netperf        |1_x_netclient->1_x_netserver                      |73.69.10.139    |1358912347     |none                                    |6B1F9B2D-919A-5A3F-ACC4-40B822C812F5
|ai_15     |netperf        |1_x_netclient->1_x_netserver                      |155.168.186.203 |1358912349     |none                                    |9CF7E55B-DF20-57A6-8097-84B2BF8658F6
|ai_16     |netperf        |1_x_netclient->1_x_netserver                      |110.78.115.121  |1358912351     |none                                    |4EAF5749-25D5-5DFF-99E3-7454E2218DA1
|ai_17     |netperf        |1_x_netclient->1_x_netserver                      |27.57.151.226   |1358912353     |none                                    |814D07E4-E0A4-5DB6-B7E6-BE1422E9B3EE
|ai_18     |specweb        |1_x_client_specweb->1_x_specwebfront->1_x_specwebback|6.3.244.224     |1358912358     |none                                    |B767323F-7BBA-51DF-965C-0AC3AC24EA69
|ai_19     |tradelite      |1_x_client_tradelite->1_x_was                     |172.195.224.119 |1358912361     |none                                    |7A04FB27-41CB-56BF-8C7A-97E564C7E108
|ai_20     |hpcc           |1_x_fen_hpc->3_x_cn_hpc                           |79.143.111.53   |1358912364     |none                                    |F78136C2-A67C-5D2C-8A6E-C67EDBD28789

The following VMs are attached to this experiment (Cloud TESTCLOUD) :
|name          |role                |size        |cloud_ip        |host_name                  |vmc_pool            |ai      |aidrs      |svm_stub_vmc     |uuid
|vm_33         |client_daytrader    |nano32      |141.98.169.121  |simhostd1                  |LG                  |ai_3    |none       |none             |B3C70489-FAA9-51AC-93D3-4CD8824F4C4E
|vm_35         |db2                 |copper32    |6.33.25.203     |simhostc2                  |SUT                 |ai_3    |none       |none             |7EA11E39-05B7-55AC-9DCF-859A3ED6B774
|vm_34         |was                 |iron32      |122.233.44.250  |simhostb1                  |SUT                 |ai_3    |none       |none             |C5A52973-885F-5E10-BB14-312B7C7A42EC
|vm_36         |client_daytrader    |nano32      |20.62.246.201   |simhostd0                  |LG                  |ai_4    |none       |none             |5B0F2AA5-F9AD-59EA-BC73-1158F3CE045A
|vm_38         |db2                 |copper32    |94.33.53.77     |simhostc0                  |SUT                 |ai_4    |none       |none             |1D4ACB1F-496B-5AD7-AC7B-F48747946C30
|vm_37         |was                 |iron32      |209.154.30.29   |simhosta0                  |SUT                 |ai_4    |none       |none             |61D04C3E-05A5-58A9-90A7-0828A592F247
|vm_39         |netclient           |platinum64  |88.182.228.241  |simhosta1                  |SUT                 |ai_5    |none       |none             |20413B55-E05D-5FE7-BE95-75BB5E3B3EAA
|vm_40         |netserver           |iron32      |179.252.55.59   |simhostb4                  |SUT                 |ai_5    |none       |none             |E2836789-BFD8-56CC-89C2-515EDC558C26
|vm_48         |lb                  |iron32      |10.177.191.53   |simhostb2                  |SUT                 |ai_8    |none       |none             |83D03AFF-D1C0-5B18-84FA-8AC0F0E4E141
|vm_47         |client_daytrader    |nano32      |120.99.243.188  |simhostd3                  |LG                  |ai_8    |none       |none             |4ABA46E4-AA10-5094-BBA9-0F2D7DF457B0
|vm_50         |was                 |iron32      |239.137.217.196 |simhostc1                  |SUT                 |ai_8    |none       |none             |A8F030D6-28EE-598F-920B-BFB308B2FFA4
|vm_49         |was                 |iron32      |38.98.80.56     |simhostc0                  |SUT                 |ai_8    |none       |none             |AB15F359-12EB-529E-ABC2-5A1D520E4AA5
|vm_51         |db2                 |gold32      |193.150.194.155 |simhosta3                  |SUT                 |ai_8    |none       |none             |9F49A85B-F382-56DD-8C4A-A9FF15894397
|vm_52         |client_daytrader    |nano32      |88.118.248.29   |simhostd3                  |LG                  |ai_9    |none       |none             |68465B66-D241-5522-878F-F9C38DFDFBF2
|vm_53         |was                 |iron32      |79.105.170.135  |simhostc0                  |SUT                 |ai_9    |none       |none             |7DDD2C72-498A-5216-9AFF-F46656E5DB8C
|vm_54         |db2                 |copper32    |105.2.132.123   |simhostc4                  |SUT                 |ai_9    |none       |none             |870B0B43-85AC-5DEC-A725-5F057A1DF9F6
|vm_55         |client_daytrader    |nano32      |112.27.118.248  |simhostd3                  |LG                  |ai_10   |none       |none             |B3147FAC-13F6-57E8-B1E3-8D6C2C9F6C71
|vm_56         |was                 |iron32      |24.149.93.124   |simhostc0                  |SUT                 |ai_10   |none       |none             |457C0312-7579-52BF-A901-7349AE435981
|vm_57         |db2                 |copper32    |44.239.140.109  |simhostb3                  |SUT                 |ai_10   |none       |none             |B563AF56-FD90-5545-AEE4-A1E42A62E40A
|vm_58         |client_daytrader    |nano32      |127.64.103.51   |simhostd3                  |LG                  |ai_11   |none       |none             |6EC08E6D-952F-5A24-97AB-11CC49E55BFB
|vm_59         |was                 |iron32      |118.3.19.218    |simhostc1                  |SUT                 |ai_11   |none       |none             |BD51BD8C-EDFC-5847-8357-85D22499CD94
|vm_60         |db2                 |copper32    |198.69.239.253  |simhostc1                  |SUT                 |ai_11   |none       |none             |F6FD7741-9047-5CDF-870F-AAE9508CAEB4
|vm_61         |client_daytrader    |nano32      |192.201.187.120 |simhostd0                  |LG                  |ai_12   |none       |none             |44380C9F-1399-55DD-954F-7CC55A49965C
|vm_63         |was                 |iron32      |32.74.70.10     |simhostb4                  |SUT                 |ai_12   |none       |none             |83A272D4-B991-5B5C-8C63-D3F38CB7E013
|vm_62         |lb                  |iron32      |219.206.192.90  |simhostc3                  |SUT                 |ai_12   |none       |none             |F799F109-87B5-508C-9CEC-29B6FF051041
|vm_64         |was                 |iron32      |160.94.104.59   |simhosta0                  |SUT                 |ai_12   |none       |none             |1AD4E8D6-4012-5192-9EC2-1462A5E0F97F
|vm_65         |db2                 |gold32      |29.22.190.28    |simhosta3                  |SUT                 |ai_12   |none       |none             |9B158893-FD0B-515E-B819-F7556B3BDD5C
|vm_67         |lb                  |iron32      |57.59.239.15    |simhostc4                  |SUT                 |ai_13   |none       |none             |65330F03-E82D-52C4-B4A1-30D2101E9106
|vm_68         |was                 |iron32      |115.236.214.209 |simhostc0                  |SUT                 |ai_13   |none       |none             |FAC5B672-693D-5F96-968A-B4D8599AE938
|vm_66         |client_daytrader    |nano32      |243.19.9.98     |simhostd1                  |LG                  |ai_13   |none       |none             |F7E06EA0-14CA-546D-98DB-FED09FD2D5A0
|vm_69         |was                 |iron32      |219.150.160.180 |simhostb3                  |SUT                 |ai_13   |none       |none             |319BFD02-4625-5B5C-A012-B438F1BC1F96
|vm_70         |db2                 |gold32      |102.92.219.203  |simhostc1                  |SUT                 |ai_13   |none       |none             |79F7FCAD-8B56-5121-9F2A-751115A3ABBA
|vm_71         |netclient           |iron32      |73.69.10.139    |simhostd1                  |LG                  |ai_14   |none       |none             |5DDE12FC-0676-59DC-A8E8-E12F5D7293AF
|vm_72         |netserver           |iron32      |14.226.76.66    |simhostc4                  |SUT                 |ai_14   |none       |none             |9FA646A5-9127-5F55-B7EC-BFCCEBC6A97D
|vm_73         |netclient           |iron32      |155.168.186.203 |simhostd4                  |LG                  |ai_15   |none       |none             |2D0FF75D-3B0D-5282-BD79-42E084448408
|vm_74         |netserver           |iron32      |111.41.87.37    |simhosta2                  |SUT                 |ai_15   |none       |none             |2A254360-851B-516E-AF22-B0447955E5F6
|vm_75         |netclient           |iron32      |110.78.115.121  |simhostd2                  |LG                  |ai_16   |none       |none             |C1BAEAF7-3C44-574A-AB27-372FAB33090C
|vm_76         |netserver           |iron32      |4.60.249.79     |simhostc2                  |SUT                 |ai_16   |none       |none             |EE16EAF8-2EF6-5E50-B8D1-EC328F12889F
|vm_77         |netclient           |iron32      |27.57.151.226   |simhostd4                  |LG                  |ai_17   |none       |none             |F14AD73F-8E63-5A46-BC7C-48CC05C95D2F
|vm_78         |netserver           |iron32      |250.210.80.219  |simhosta1                  |SUT                 |ai_17   |none       |none             |BD7DB2F3-0AD5-5077-8838-0A237AA55816
|vm_79         |client_specweb      |micro32     |6.3.244.224     |simhostd3                  |LG                  |ai_18   |none       |none             |158918DF-EB44-57CB-9860-FDAFBD42DCFE
|vm_81         |specwebback         |platinum64  |254.131.182.177 |simhosta2                  |SUT                 |ai_18   |none       |none             |77AC9488-D4FC-5640-A48D-7FDD2B8F1CFE
|vm_80         |specwebfront        |copper32    |189.89.31.163   |simhostc4                  |SUT                 |ai_18   |none       |none             |2EA34557-46FE-5C24-BFA7-CF218C0B61AF
|vm_82         |client_tradelite    |nano32      |172.195.224.119 |simhostd3                  |LG                  |ai_19   |none       |none             |D8744A89-DDDA-581A-8EB8-223D691ED804
|vm_83         |was                 |iron32      |20.238.123.77   |simhostc1                  |SUT                 |ai_19   |none       |none             |A0693F05-0395-5F2C-943D-23E49640545E
|vm_85         |cn_hpc              |bronze32    |14.225.133.238  |simhostb0                  |SUT                 |ai_20   |none       |none             |F9D798BA-E81B-5214-916C-15CECED1FD1F
|vm_84         |fen_hpc             |bronze32    |79.143.111.53   |simhostc3                  |SUT                 |ai_20   |none       |none             |1E720B0B-34A7-554D-BC4D-DA2DFF2BA5BD
|vm_86         |cn_hpc              |bronze32    |73.215.44.132   |simhosta2                  |SUT                 |ai_20   |none       |none             |542751F5-612A-5728-BB28-F921D06D44AB
|vm_87         |cn_hpc              |bronze32    |190.118.85.179  |simhostb0                  |SUT                 |ai_20   |none       |none             |14D446F1-41EC-5057-9FB0-2A45459FC6F0

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |4.33M                         
|Redis Version                                        |2.6.7                         
|Uptime (in seconds)                                  |459681                        
|Total Connections Received                           |2828                          
|Total Commands Processed                             |4330516                       
|Number of Keys                                       |858                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.2.2                         
|Storage Size                                         |1179648                       
|Data Size                                            |285684                        
|Index Size                                           |294336                        
|Average Object Size                                  |485.857142857                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |49                            
|SVMs                                                 |0                             
|AIs                                                  |16                            
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |49                            
|VMs ARRIVED                                          |84                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |35                            
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |3                             
|SVM RESERVATIONS                                     |0                             
|SVMs ARRIVED                                         |0                             
|SVMs ARRIVING                                        |0                             
|SVMs DEPARTED                                        |0                             
|SVMs DEPARTING                                       |0                             
|SVMs FAILED                                          |0                             
|AI RESERVATIONS                                      |16                            
|AIs ARRIVED                                          |20                            
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |4                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |234                           

 ---------------- VMC-WIDE COUNTERS ----------------
|66C80F72-DB30-5A63-B169-31C6A86D021F (simzone_c) VM RESERVATIONS|18                            
|D3E36C93-1427-5D86-BC91-CB0FA4A8E918 (simzone_d) VM RESERVATIONS|14                            
|FEBD66A7-E012-534C-A431-00CE123E50EF (simzone_a) VM RESERVATIONS|9                             
|9859398A-822B-50DC-8C37-7E3C5CBAE602 (simzone_b) VM RESERVATIONS|8                             

